packages/wonder-blocks-tooltip/components/tooltip-popper.js

```diff
@@ -37,14 +29,43 @@ type Props = {|
 |};
 
 export default class TooltipPopper extends React.Component<Props> {
-    _renderPositionedContent(childrenProps: PopperJSChildrenProps) {
-        const {children, placement} = this.props;
+    _bubbleRefTracker = new RefTracker();
+    _tailRefTracker = new RefTracker();
+
+    _getUpdatedBubbleRefTracker(targetRefFn: (?HTMLElement) => void) {
```

Comment:
Where is this called?

Reply:
I don't think it is anymore. I'll delete it.